lib/perl/Genome/Model/Tools/EpitopePrediction/NewPipeline.pm

```diff
@@ -0,0 +1,192 @@
+package Genome::Model::Tools::EpitopePrediction::NewPipeline;
```

Comment:
Will this still be the `NewPipeline` in five years?  (Not that I have better names to suggest!)